packages/collector/src/logger.js

```diff
@@ -42,17 +96,26 @@ exports.init = function init(config, isReInit) {
     //      We cannot fix this in our side.
     //      https://github.com/winstonjs/winston/issues/1854#issuecomment-710195110
     parentLogger = config.logger.child({
-      module: 'instana-nodejs-logger-parent'
+      // TODO: Rename to "instana-nodejs-logger" in next major version.
+      module: 'instana-nodejs-logger-parent',
+      threadId
     });
   } else if (config.logger && hasLoggingFunctions(config.logger)) {
     // A custom non-bunyan/non-pino logger has been provided via config. We use it as is.
     parentLogger = config.logger;
   } else {
-    // No custom logger has been provided via config, we create a new pino logger as the parent logger for all loggers
-    // we create later on.
+    const os = require('os');
+
+    // No custom logger has been provided via config, we create a new pino logger as our internal
+    // Instana logger.
+
+    // If we override `base`, we have to provide the default pino values too.
+    // Pino does not merge them together.
+    // See https://github.com/pinojs/pino/blob/main/docs/api.md#base-object
     parentLogger = uninstrumentedLogger({
       name: '@instana/collector',
-      level: 'info'
+      level: 'info',
+      base: { threadId, pid: process.pid, hostname: os.hostname() }
```

Comment:
The new test I have added in #1564 tests this logic now automatically.
Because earlier we attached the threadId on the child!